doc/api/process.md

```diff
@@ -1342,6 +1342,12 @@ The `process.getgroups()` method returns an array with the supplementary group
 IDs. POSIX leaves it unspecified if the effective group ID is included but
 Node.js ensures it always is.
 
+```js
+if (process.getgroups) {
+  console.log(`Current groups: ${process.getgroups()}`);
```

Comment:
```suggestion
  console.log(process.getgroups()); // [ 16, 21, 297 ]
```

Reply:
@Trott - I understand your point and thanks for the review. I have followed your suggestion. Please have a look.